### PR TITLE
Changes to be applied for 8.0.0 release

### PR DESCRIPTION
**Pull request by scarletfog**
A WIP

A basic checklist to mark which parts of the documentation were reviewed.
Some parts are in bold - I think I should pay extra attention to them.

Changes are being added here: https://github.com/handsontable/docs/tree/draft-next-ania

- [x] Getting started
- [ ] Basic usage 
  - [x] Quick start
  - [ ] Data binding
  - [ ] Data sources
  - [x] Load and save
  - [x] Setting options
  - [x] Grid sizing
  - [x] Using callbacks - `hooks = Handsontable.hooks.getRegistered();` ?
  - [x] Keyboard navigation
  - [x] Internationalization
- [ ] Developer guide
  - [x] Custom build
  - [x] Custom plugin (are skeletons still valid?)
  - [x] Cell types
  - [x] Cell editor - `this.TEXTAREA.className = 'handsontableInput';`? (very long section btw)
  - [x] Cell function
  - [x] Testing
  - [x] Performance tips
  - [x] Release notes
- [x] Community
**WRAPPERS**
- [x] Wrapper for React
  - [x] Installation
  - [x] Basic examples
  - [x] Using the HotColumn component
  - [x] Custom Context menu example - very laggy page
  - [x] Custom editor example ` this.TEXTAREA.className = 'handsontableInput';`?
  - [x] Custom renderer example
  - [x] Language change example
  - [x] Redux example
  - [x] Referencing the Handsontable instance
- [x] Wrapper for Angular
  - [x] Installation
  - [x] Basic example
  - [x] Custom ID
  - [x] Custom Context Menu example
  - [x] Custom editor example
  - [x] Custom renderer example
  - [x] Language change example
  - [x] Referencing the Handsontable instance
- [ ] Wrapper for Vue
  - [ ] Installation
  - [ ] Basic example
  - [ ] Using the hot-column component
  - [ ] Custom ID, Class, Style and other attributes
  - [ ] Custom Context Menu example
  - [ ] Custom editor example
  - [ ] Custom renderer example
  - [ ] Language change example
  - [ ] Vuex example
  - [ ] Referencing the Handsontable instance
**DEMOS**
- [ ] **Rows and columns**
- [ ] **Data operations**
- [ ] Cell features
- [ ] Cell types
- [ ] Utilities
- [ ] Appearance 
- [ ] **Experimental features**
**API REFERENCE**
Core
- [ ] Core
- [ ] **Hooks**
- [ ] Options
Plugins
- [ ] AutoColumnSize
- [ ] AutoRowSize
- [ ] Autofill
- [ ] BindRowsWithHeaders
- [ ] CollapsibleColumns
- [ ] **ColumnSorting**
- [ ] ColumnSummary
- [ ] Comments
- [ ] **ContextMenu**
- [ ] CopyPaste
- [ ] **CustomBorders**
- [ ] DragToScroll
- [ ] DropdownMenu
- [ ] ExportFile
- [ ] Filters
- [ ] **HeaderTooltips**
- [ ] **HiddenColumns**
- [ ] **HiddenRows**
- [ ] ManualColumnFreeze
- [ ] ManualColumnMove
- [ ] ManualColumnResize
- [ ] ManualRowMove
- [ ] ManualRowResize
- [ ] MergeCells
- [ ] MultiColumnSorting
- [ ] **NestedHeaders**
- [ ] **ObserveChanges**
- [ ] PersistentState
- [ ] Search
- [ ] **TrimRows**
- [ ] UndoRedo
**EXPERIMENTAL - check if this was removed, should have been.**

### CODE REVIEW

File: tutorials/features.html [hunk 1]
```diff
@@ -205,24 +205,6 @@
...
-  <table class="static-data features-table">
-    <thead>
-    <tr>
-      <th>Name</th>
-      <th>Description</th>
-    </tr>
-    </thead>
-    <tbody>
-      <tr>
-        <td><a href="/docs/<?js= version ?>/demo-formula-support.html">Formulas</a></td>
```
Comment (wojciechczerniak) [inline tutorials/features.html:218]: We're removing Formulas Plugin?

Comment (scarletfog) [reply tutorials/features.html:218]: Wtf me.
WIll restore asap 🤦

**Comment by scarletfog** [comment]
My bad @aninde I restored the wrong part.

**Comment by scarletfog** [comment]
I managed to go through some parts only before 🌴

File: tutorials/collapsing-columns.html [hunk 2]
```diff
@@ -15,9 +15,6 @@ <h3 id="overview">Overview</h3>
       property defined.<br/>
       The plugin adds a "expand/collapse" button multi-column headers (created with the Nested Headers plugin).
       Clicking these buttons will collapse (or expands) all "child" headers, leaving the first one visible.<br/>
-      <br/>
-      <strong>Node:</strong> You'll need to enable the <a href="/docs/<?js= version ?>/demo-nested-headers.html">Nested Headers</a> and
-      <a href="/docs/<?js= version ?>/demo-hiding-columns.html">Hidden Columns</a> plugins in order for this plugin to work properly.
```
Comment (wojciechczerniak) [inline tutorials/collapsing-columns.html:20]: Nested Headers are still needed AFAIK.

File: tutorials/compatibility.html [hunk 3]
```diff
@@ -33,6 +33,7 @@
           <li>Flickering headers</li>
         </ul>
     </div>
+    <p><strong>Note: Support for Internet Explorer is deprecated in 8.0.0 version.</strong></p>
```
Comment (budnix) [inline tutorials/compatibility.html:36]: The deprecation is not postponed? @wojciechczerniak

Comment (wojciechczerniak) [reply tutorials/compatibility.html:36]: Removing support is, with v8 we just inform about our intention to drop the support somehwere in the future.

**Comment by aninde** [comment]
`observeChanges` is deprecated but it still, appear on every draft branch. Is that planned?
https://github.com/handsontable/docs/blob/951104f5666564c946597f2f6184adf4334066c7/tutorials/using-callbacks.html#L196
https://github.com/handsontable/docs/blob/444e33bdaa22ce1e88dce363332cd1788fe66c26/tutorials/using-callbacks.html#L196
https://github.com/handsontable/docs/blob/da637270f715f33dcc649fafe08ed117bef33c14/tutorials/using-callbacks.html#L196

File: tutorials/compatibility.html (hunk 3 above)
Comment (budnix) [reply tutorials/compatibility.html:36]: I'd change the sentence meaning 🤔  Now it literally suggests that v8 won't get any updates for IE when something bad will happen.

File: tutorials/moving.html [hunk 4]
```diff
@@ -22,27 +29,56 @@
...
+        Some of actions to be performed by <code>moveRows</code> function aren't possible, i.e. we can't move more than one element to the last position. In this case, the move will be cancelled.
+        The Plugin's <code><a href="/ManualRowMove.html#isMovePossible">isMovePossible</a></code> API method and the <code>movePossible</code> parameter of
+        <code>beforeRowMove</code> and <code>afterRowMove</code> hooks may be helpful in determining such situations.
+      </p>
     </div>
   </div>
   <p class="gap-top-xsmall">
-    <a href="https://github.com/handsontable/docs/edit/<?js= version ?>/tutorials/moving.html" class="edit-doc" target="_blank">
-      Edit this page
+    <a href="https://github.com/handsontable/docs-pro/edit/<?js= version ?>/tutorials/moving.html" class="edit-doc" target="_blank">
```
Comment (budnix) [inline tutorials/moving.html:80]:
```suggestion
    <a href="https://github.com/handsontable/docs/edit/<?js= version ?>/tutorials/moving.html" class="edit-doc" target="_blank">
```

File: tutorials/moving.html [hunk 5]
```diff
@@ -22,27 +29,56 @@
...
-        });</script>
+        });
+      </script>
+    </div>
+
+    <div class="example-container clearfix" name="last">
+      <h3 id="manualRowMove">Drag and move actions of <code>manualRowMove</code> plugin</h3>
+      <p>There are significant differences between the plugin's <code>dragRows</code> and <code>moveRows</code> API functions. Both of them change the order of rows,
+        but what is important, they rely on other kind of indexes. The differences between them are shown in the diagrams below. Please keep in mind that both of these methods trigger the
+        <code><a href="/Hooks.html#event:afterRowMove">afterRowMove</a></code> and <code><a href="/Hooks.html#event:beforeRowMove">beforeRowMove</a></code> hooks,
```
Comment (budnix) [inline tutorials/moving.html:54]:
```suggestion
        <code><a href="/docs/<?js= version ?>Hooks.html#event:afterRowMove">afterRowMove</a></code> and <code><a href="/docs/<?js= version ?>/Hooks.html#event:beforeRowMove">beforeRowMove</a></code> hooks,
```

File: tutorials/moving.html [hunk 6]
```diff
@@ -22,27 +29,56 @@
...
+        but only <code>dragRows</code> passes the <code>dropIndex</code> argument to them.
+      </p>
+
+      <p id="dragRows">
+        1. The <code><a href="/ManualRowMove.html#dragRows">dragRows</a></code> method with the <code>dropIndex</code> parameter. This argument points to where we are going to drop the moved elements.
```
Comment (budnix) [inline tutorials/moving.html:59]:
```suggestion
        1. The <code><a href="/docs/<?js= version ?>ManualRowMove.html#dragRows">dragRows</a></code> method with the <code>dropIndex</code> parameter. This argument points to where we are going to drop the moved elements.
```

File: tutorials/moving.html [hunk 7]
```diff
@@ -22,27 +29,56 @@
...
+      </p>
+      <p>
+        <img src="/images/drag_action.svg" alt="dragRows method" />
+      </p>
+
+      <p id="moveRows">
+        2. The <code><a href="/ManualRowMove.html#moveRows">moveRows</a></code> method with the <code>finalIndex</code> parameter.
```
Comment (budnix) [inline tutorials/moving.html:66]:
```suggestion
        2. The <code><a href="/docs/<?js= version ?>/ManualRowMove.html#moveRows">moveRows</a></code> method with the <code>finalIndex</code> parameter.
```

File: tutorials/moving.html [hunk 8]
```diff
@@ -22,27 +29,56 @@
...
+        The argument points to where the elements will be placed after the <i>moving</i> action (<code>finalIndex</code> being the index of the first moved element).
+      </p>
+      <p>
+        <img src="/images/move_action.svg" alt="moveRows method" />
+      </p>
+      <p>
+        Some of actions to be performed by <code>moveRows</code> function aren't possible, i.e. we can't move more than one element to the last position. In this case, the move will be cancelled.
+        The Plugin's <code><a href="/ManualRowMove.html#isMovePossible">isMovePossible</a></code> API method and the <code>movePossible</code> parameter of
```
Comment (budnix) [inline tutorials/moving.html:74]:
```suggestion
        The Plugin's <code><a href="/docs/<?js= version ?>/ManualRowMove.html#isMovePossible">isMovePossible</a></code> API method and the <code>movePossible</code> parameter of
```

File: tutorials/moving.html [hunk 9]
```diff
@@ -22,27 +29,56 @@
...
+        but what is important, they rely on other kind of indexes. The differences between them are shown in the diagrams below. Please keep in mind that both of these methods trigger the
+        <code><a href="/Hooks.html#event:afterRowMove">afterRowMove</a></code> and <code><a href="/Hooks.html#event:beforeRowMove">beforeRowMove</a></code> hooks,
+        but only <code>dragRows</code> passes the <code>dropIndex</code> argument to them.
+      </p>
+
+      <p id="dragRows">
+        1. The <code><a href="/ManualRowMove.html#dragRows">dragRows</a></code> method with the <code>dropIndex</code> parameter. This argument points to where we are going to drop the moved elements.
+      </p>
+      <p>
+        <img src="/images/drag_action.svg" alt="dragRows method" />
```
Comment (budnix) [inline tutorials/moving.html:62]:
```suggestion
        <img src="/docs/images/drag_action.svg" alt="dragRows method" />
```

File: tutorials/moving.html [hunk 10]
```diff
@@ -22,27 +29,56 @@
...
+      </p>
+
+      <p id="moveRows">
+        2. The <code><a href="/ManualRowMove.html#moveRows">moveRows</a></code> method with the <code>finalIndex</code> parameter.
+        The argument points to where the elements will be placed after the <i>moving</i> action (<code>finalIndex</code> being the index of the first moved element).
+      </p>
+      <p>
+        <img src="/images/move_action.svg" alt="moveRows method" />
```
Comment (budnix) [inline tutorials/moving.html:70]:
```suggestion
        <img src="/docs/images/move_action.svg" alt="moveRows method" />
```

**Comment by wszymanski** [comment]
> Has `Options.html` been updated? It still contains information on the deprecated `observeChanges` without mentioning it being deprecated.

I will do separate PR for this, as this should be changed in Handsontable code.

> Additionally, `observeChanges` is still used on
> https://github.com/handsontable/docs/blob/951104f5666564c946597f2f6184adf4334066c7/tutorials/using-callbacks.html#L196

For me, it should be here, as plugin is just deprecated, not removed. It seems that documentation for all deprecated items is preserved until they are removed.

> Is that a valid name since `modifyRow` was removed?
> https://github.com/handsontable/docs/blob/951104f5666564c946597f2f6184adf4334066c7/tutorials/using-callbacks.html#L151

Done. Thanks.

**Pull request merged**